.eslintrc.yaml

```diff
@@ -1,11 +1,25 @@
-extends: '@commercetools-frontend/eslint-config-mc-app'
+extends:
+  - xo
+  - '@commercetools-frontend/eslint-config-mc-app'
+  - xo-react
+  - xo-typescript
+  - prettier/@typescript-eslint
```

Comment:
Maybe worth adding `prettier/react`? Not sure.

Reply:
I think this part here is what we need to fix, as we are extending and overwriting a lot of presets. For instance, that's why our own preset is in between the others, otherwise we get more conflicts.

I think we need to either include the typescript stuff in our preset (so that we can determine the proper order of the presets) or have a separate one tailored to TS.

WDYT?

Reply:
We should then mix/make our own preset as a package `mc-preset-typescript` or something are carefully build up the preset there from other building blocks?

Reply:
Yeah, I don't have a good idea at the moment. I was hoping you do 😇